lib/retriable/config.rb

```diff
@@ -1,29 +1,48 @@
+require_relative "exponential_backoff"
+
 module Retriable
   class Config
-    attr_accessor :sleep_disabled
-    attr_accessor :tries
-    attr_accessor :base_interval
-    attr_accessor :max_interval
-    attr_accessor :rand_factor
-    attr_accessor :multiplier
-    attr_accessor :max_elapsed_time
-    attr_accessor :intervals
-    attr_accessor :timeout
-    attr_accessor :on
-    attr_accessor :on_retry
+    ATTRIBUTES = [
+      :tries,
+      :base_interval,
+      :max_interval,
+      :rand_factor,
+      :multiplier,
+      :sleep_disabled,
+      :max_elapsed_time,
+      :intervals,
+      :timeout,
+      :on,
+      :on_retry,
+    ].freeze
```

Comment:
i do think it makes more sense to do
```ruby
ATTRIBUTES = ExponentialBackoff::ATTRIBUTES + [(the rest of the attributes)
```
just to avoid code duplication